python_agent_utils/messages/fields.py

```diff
@@ -0,0 +1,497 @@
+import ipaddress
+import json
+from abc import ABCMeta, abstractmethod
+from typing import Optional, Iterable
+
+import base58
+import dateutil.parser
```

Comment:
Is this a newly introduced dependency that hasn't made it into `requirements.txt`?
```
tests/__init__.py:12: in <module>
    from python_agent_utils.messages.message import Message
../python_agent_utils/messages/message.py:11: in <module>
    from python_agent_utils.messages.fields import NonNegativeNumberField, MapField, DIDField, ISODatetimeStringField
../python_agent_utils/messages/fields.py:7: in <module>
    import dateutil.parser
E   ModuleNotFoundError: No module named 'dateutil'
```

Reply:
Added